File: X01/x01.c
#include <stdio.h>

int main(void) {
    int num_unit, unit_length;
    char buf[1000];

    fgets(buf, sizeof(buf), stdin);
    sscanf(buf, "%d\n", &num_unit);
    unit_length = num_unit * 25;
    printf("%d\n", unit_length);
    return 0;
}

File: X01/x01.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	sc := bufio.NewScanner(os.Stdin)
	sc.Scan()

	num_unit, _ := strconv.Atoi(sc.Text())
	unit_length := num_unit * 25
	fmt.Println(unit_length)
}
